### PR TITLE
Implement 'Last updates timestamp' feature #98

### LINKED ISSUE #98: Implement 'Last updates timestamp' feature
Footer at bottom left "Last updated: [not implemented yet]"

### PR DESCRIPTION
**Pull request by mauris**
Implementation takes advantage of the EventsCenter to check for updates to AddressBook

however when it starts up, the app is not aware of the address book's last updated (before the app starts up) date/time. 

Fixes #98

### CODE REVIEW

File: src/main/java/seedu/address/ui/StatusBarFooter.java [hunk 1]
```diff
@@ -81,4 +88,8 @@ public String getFxmlPath() {
         return FXML;
     }
 
+    @Subscribe
+    public void handleAddressBookChangedEvent(AddressBookChangedEvent mce) {
```
Comment (damithc) [inline src/main/java/seedu/address/ui/StatusBarFooter.java]: `mce` --> `abce`? (change it in the other place as well).

File: src/main/java/seedu/address/ui/StatusBarFooter.java [hunk 2]
```diff
@@ -36,7 +43,7 @@ public static StatusBarFooter load(Stage stage, AnchorPane placeHolder, String s
     public void configure(String saveLocation) {
         addMainPane();
         addSyncStatus();
-        setSyncStatus("Last updated: [not implemented yet]");
+        setSyncStatus("Ready");
```
Comment (damithc) [inline src/main/java/seedu/address/ui/StatusBarFooter.java]: `Unknown` or `Not updated yet in this session`?

File: src/main/java/seedu/address/ui/StatusBarFooter.java (hunk 1 above)
Comment (mauris) [inline src/main/java/seedu/address/ui/StatusBarFooter.java]: renamed instances of `mce` to `abce`.

File: src/main/java/seedu/address/ui/StatusBarFooter.java (hunk 2 above)
Comment (mauris) [inline src/main/java/seedu/address/ui/StatusBarFooter.java]: chose `Not updated yet in this session`. We can also choose to read in the last updated from file system.

**Pull request merged**